src/routes/bills/[id]/+page.svelte

```diff
@@ -1,7 +1,362 @@
+										{:else if mergedBill.proposerType === BillProposerType.People && mergedBill.proposedByPeople}
+											{mergedBill.proposedByPeople.ledBy +
+												' และประชาชน' +
+												mergedBill.proposedByPeople.signatoryCount +
+												' คน'}
+										{/if}
+									</span>
+								</li>
+							{/each}
+						</ul>
+					</div>
+				{/if}
+			</div>
+			<div class="flex flex-col gap-2 md:w-56">
+				{#if bill.attachment}
+					<DownloadData links={[{ label: bill.attachment.label, url: bill.attachment.url }]} />
+				{/if}
+				<p class="label-01 text-text-02">
+					อัปเดตข้อมูล: {new Date().toLocaleDateString('th-TH', {
+						year: 'numeric',
+						month: 'short',
+						day: 'numeric'
+					})}
+				</p>
+				<!-- TODO: add link -->
+				<a href="/" class="mr-auto helper-text-01 underline"> ที่มาและข้อจำกัดข้อมูล </a>
+				<Share label="แชร์มติ" />
+			</div>
+		</div>
+	</section>
+
+	{#if bill.proposerType === BillProposerType.Politician && bill.proposedLedByPolitician && bill.coProposedByPoliticians}
+		<section title="List of legal proponents" class="px-4 py-8 md:px-16 md:py-12">
+			<div class="flex flex-col gap-5">
+				<h1 class="fluid-heading-04 text-text-primary">รายชื่อผู้เสนอกฏหมาย</h1>
+				<hr class="border-gray-30" />
+				<div class="flex flex-col gap-7 md:flex-row">
+					<div class="flex flex-col gap-3 md:w-1/3">
+						<b class="handing-02 text-text-primary">ผู้เสนอ</b>
+						<PoliticianProfile
+							id={bill.proposedLedByPolitician.id}
+							firstname={bill.proposedLedByPolitician.firstname}
+							lastname={bill.proposedLedByPolitician.lastname}
+							avatar={bill.proposedLedByPolitician.avatar}
+							party={bill.proposedLedByPolitician.partyRoles.find((e) => !e.endedAt)?.party}
+							role={bill.proposedLedByPolitician.assemblyRoles.find((e) => !e.endedAt)?.role + ''}
+						/>
+					</div>
+					<div class="flex flex-col gap-3 md:w-full">
+						<p class="body-02 text-text-02">
+							<b class="handing-02 text-text-primary">ผู้ร่วมเสนอ</b>
+							{bill.coProposedByPoliticians.length} คน
+						</p>
+						<div class="flex flex-col flex-wrap md:flex-row gap-3 px-3">
+							{#if partiescoProposed}
+								{#each Object.entries(partiescoProposed) as [party, politicians] (party)}
+									<CoPartyProposer {party} {politicians} />
+								{/each}
+							{/if}
+						</div>
+						<div class="flex flex-col gap-2 px-5">
+							<p class="label-01 text-text-02 mr-2">เรียงตามตัวอักษร</p>
+							<div class="relative flex flex-col">
+								<table class="w-full">
+									{#each bill.coProposedByPoliticians as politician, i}
+										{#if i < 8}
+											<CoProposer
+												index={i + 1}
+												logo={politician.partyRoles.find((e) => !e.endedAt)?.party.logo}
+												firstname={politician.firstname}
+												lastname={politician.lastname}
+											/>
+										{/if}
+									{/each}
+								</table>
+
+								{#if bill.coProposedByPoliticians.length > 8}
```

Comment:
We should avoid magic number, people won't understand what is it, and if they want to change, how many places they need to change. So better declared it in `const` first like
```ts
const MAX_DISPLAY_COPROPOSER = 8;
```